packages/mobile/src/web3/saga.ts

```diff
@@ -455,71 +454,71 @@ export function* switchToGethFromZeroSync() {
     }
 
     yield spawn(gethSaga)
-
     switchWeb3ProviderForSyncMode(false)
-    // Ensure web3 is fully synced using new provider
+    yield take(GethActions.SET_GETH_CONNECTED)
```

Comment:
Wait for geth to be connected before starting web3 sync

Reply:
Should we use `waitForGethConnectivity` here like this line: https://github.com/celo-org/celo-monorepo/blob/master/packages/mobile/src/web3/saga.ts#L133

Reply:
Yes, updated!